### DISCUSSION TITLE
Suddenly getting error in CI with Remix

### DISCUSSION THREAD

**Discussion by candidia** [vercel/community#1604] [Help]
### Summary

I am getting `Error: File /vercel/path0/api/index.js does not exist.` in CI even if I go back to a commit that had a green build and make a comment and push again. This started happening out of nowhere and I haven't changed anything. Anyone have an idea?

**Comment by ehwarren** [comment]
Same here. Last time this happened they recommended I roll back to an older CLI version..

Setting environment variable `VERCEL_CLI_VERSION=vercel@28.16.4` seems to do the trick as a **Temporary Solution**

Looks like they broke something again!

**Reply by candidia** [reply]
Thank you for the temp solution!

**Reply by frontsideair** [reply]
Workaround indeed seems to work, thanks!

**Reply by ehwarren** [reply]
https://github.com/vercel/vercel/releases/tag/vercel%4028.16.6

Tested and working with their new release. Go ahead and remove that environment variable!

**Reply by frontsideair** [reply]
Looks like the new CLI is also broken unfortunately. It doesn't fail the build, but afterwards when I visit the deployment I get a "This Serverless Function has crashed" error with no stack trace. I'll keep using the older version until the issue is resolved.

**Reply by WesleyYue** [reply]
@frontsideair I also had the "This Serverless Function has crashed" issue with no logs. I bisected my code to find the problematic code but even though I knew which exact line of code was causing the issue, I had no idea why, but maybe the info I had can help you.

https://discord.com/channels/770287896669978684/1079829753210605578/1079829753210605578

**Reply by TooTallNate** [reply]
@WesleyYue We've identified and fixed an issue related to serverless function logs reporting, so you should be able to properly debug failing routes once again. Please let us know if you still are having any issues related to that!

**Reply by candidia** [reply]
@TooTallNate Looks like something new broke. I now get the following error and it resolves by setting the environment variable @ehwarren advised.

```
21:24:33.506 | Building Remix app in production mode...
21:24:35.439 |  
21:24:35.440 | ✘ [ERROR] Could not resolve "node:crypto"
21:24:35.440 |  
21:24:35.440 | browser-route-module:routes/[...].ts?browser:4:44:
21:24:35.440 | 4 │ import { createHmac, timingSafeEqual } from "node:crypto";
21:24:35.440 | ╵                                             ~~~~~~~~~~~~~
21:24:35.441 |  
21:24:35.441 | The package "node:crypto" wasn't found on the file system but is built into node. Are you trying to bundle for node? You can use "platform: 'node'" to do that, which will remove this error.
21:24:35.441 |  
21:24:35.441 |  
21:24:35.441 | Build failed with 1 error:
21:24:35.442 | browser-route-module:routes/[...].ts?browser:4:44: ERROR: Could not resolve "node:crypto"
21:24:35.442 | Error
21:24:35.442 | at Object.onCompileFailure (/var/task/node_modules/@remix-run/dev/dist/cli/commands.js:193:13)
21:24:35.442 | at Object.compile (/var/task/node_modules/@remix-run/dev/dist/compiler/remixCompiler.js:40:134)
21:24:35.442 | at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
21:24:35.442 | at async Object.build (/var/task/node_modules/@remix-run/dev/dist/compiler/build.js:33:3)
21:24:35.442 | at async Object.build (/var/task/node_modules/@remix-run/dev/dist/cli/commands.js:188:3)
21:24:35.442 | at async Object.run (/var/task/node_modules/@remix-run/dev/dist/cli/run.js:456:7)
21:24:35.479 | Error: Command "npm run build:prod" exited with 1
21:24:36.153 | BUILD_FAILED: Command "npm run build:prod" exited with 1
```
